diff --git a/util/pcre.h b/util/pcre.h
index 896b0bd..f0a7180 100644
--- a/util/pcre.h
+++ b/util/pcre.h
@@ -500,7 +500,7 @@ class PCRE {
   bool                report_errors_;  // Silences error logging if false
   int                 match_limit_;    // Limit on execution resources
   int                 stack_limit_;    // Limit on stack resources (bytes)
-  mutable int32_t     hit_limit_;      // Hit limit during execution (bool)
+  mutable int         hit_limit_;      // Hit limit during execution (bool)

   PCRE(const PCRE&) = delete;
   PCRE& operator=(const PCRE&) = delete;
